### DISCUSSION TITLE
Problems

### DISCUSSION THREAD

**Discussion by rieders** [inventree/InvenTree#5485] [General]
Hello
I haven't been using Inventree that long.
So I have several questions.
I created parts in a category, I accidentally created a part twice.
Unfortunately it cannot be deleted. then this error message appears.

![99](https://github.com/inventree/InvenTree/assets/5175143/abd45cde-a216-410b-b473-0af994625800)

IPN- how can I create an automatic IPN. the parts I created didn't get an IPN even though I activated create automatic ipn.
Price- how can I edit the price of parts afterwards.
I forgot to specify the price when creating the stock item.
Under part pricing, I get an error "No prices available for this part" even though I specified a price for the part.

I hope someone can help me

Greetings Andre

**Comment by SchrodingersGat** [comment]
Hey Andre, it would be really useful to break these problems out into separate discussions. There's too much to tackle in a single post here!

Let's tackle the error message - it looks like you have installed the shopify plugin, but not updated the database afterwards?

**Comment by rieders** [comment]
Many thanks for the answer.
yes i just installed the shopify plugin. I had restarted the server several times.
How can I update the SQL Light database?

Greetings Andre

**Reply by SchrodingersGat** [reply]
It should be as simple as running `invoke update` after installing the plugin
